# Resume

## Education
**PhD.** in Biomedical Science, **Northwestern University** (2023)
<br>
**BSc.** in Molecular Biology, **Loyola University Chicago** (2014)
* **Select Coursework:** Bioinformatics, Quantitative Biology, Biostatistics I/II (Statistical Survey/Regression Analysis), Mathematical Statistics I/II (Probability Theory/Statistical Inference), Machine Learning, Information Management for Data Science (SQL/database management)
* **Bootcamps:** Programming Concepts & Fundamentals, R Programming , R Tidyverse, Python Programming, Python Data Wrangling and Visualization
* **Certificates:** Management for Scientists and Engineers from Northwestern University Kellogg School of Management

## Experience
Northwestern University, Department of Microbiology-Immunology              (Chicago, IL)
Graduate Research Assistant              January 2017 – Present
* Studied the role of KSHV in HIV-associated cancers using high-throughput technologies, such as genome -wide CRISPR screening & single-cell sequencing.
* Served as the primary data scientist in support of 3 grants, including on the largest single-cell genomics study conducted at Northwestern University to-date
Loyola University Chicago, Department of Bioinformatics              Chicago, IL
Research Assistant              Aug 2103 – April 2015
* Studied the microbial diversity of the nearshow waters of am urban, lacustrine environment (Lake Michigan) using targeted (16S rRNA) and shotgun sequencing approaches
* Worked within a highly interdisciplinary team of biologists/ecologists and statisticians/computer scientists

## Skills
* Python: numpy, pandas, scikit-learn, pytorch, scipy, statsmodels, matplotlib/seaborn, biopython, cython, RPy2 (R API for Python), bs4/requests (web scraping)
* R: Tidyverse, ggplot, CARAT, and several domain-specific libraries (deseq2, edgeR, seurat, survival, etc)
* Other Languages/Tools: SQL (relational databases), bash (shell scripting, Linux, high-performance computing), regular expressions, Git
•	Areas of Knowledge: Biomedical science (molecular biology, particularly cancer biology and virology), bioinformatics, statistics, machine learning

## Projects
Dissertation: The Role of FLICE-Inhibitory Proteins in Primary Effusion Lymphoma
* Identified novel regulators of TRAIL-R1 cell death and cFLIP dependence in KSHV-associated primary effusion lymphoma using genome-wide synthetic CRISPR rescue screens
* *Published in the prestigious Cell Death & Differentiation, selected for Reader’s Monthly Choice 
### Transcriptomic Analysis of KSHV Infection
* Deployed multiple supervised (KNN, linear regression) and unsupervised (denoising autoencoder, PCA, UMAP, and Leiden clustering) techniques alongside traditional statistical inference (Chi-square, NB-GLM, and Kolmogorov-Smirnov) to single cell data for QA, data cleaning, and hypothesis formation/testing
* * Determined sample sizes necessary for stable algorithm performance via bootstrapping (yielding 33% savings in cost)
* Developed a fast Cython tool for analysis of miRNA/k-mer enrichment in bulk RNA samples (rolling hash algorithm + hypergeometric testing)
### Natural Language Processing on RPG Content
* Performed doc2vec + unsupervised clustering + semi-supervised multiclass classification to develop section-level data annotations of over 3000 pieces of fan-generated RPG content scraped from Reddit
* Performed multilabel/multiclass sequence-to-sequence classification built on BERT
### Predicting Used Car Value
* Performed ordinal (random forest) regression on the 2023 CarMax dataset to predict (R2: .94 & C-Index: .94) vehicle value on a heavily obfuscated dataset
* Deployed an interactive Dash application for model inference via Shapley values
### PiPy-AWC
* Automated water level controller for home aquaria using a Raspberry Pi
*	I developed this in part to practice software design patterns like factories, observer/callback, etc.

## Publications
1.	Kuehnle N, et al. CRISPR screens identify novel regulators of cFLIP dependency and ligand-independent, TRAIL-R1-mediated cell death. Cell Death Differ. 2023 May; 30(5):1221-1234. doi: 10.1038/s41418-023-01133-0.
2.	Kuehnle N, Gottwein E. Druggable host gene dependencies in primary effusion lymphoma. Curr Opin Virol. 2022 Sep; 56:101270. doi: 10.1016/j.coviro.2022.101270.
3.	Watkins S, Kuehnle N, et al. Assessment of a Metaviromic Dataset Generated from Nearshore Lake Michigan Lake Michigan. Marine and Freshwater Research. 2022 Nov; 67(11). doi: 10.1071/MF15172
